NASA-PDS/validate
# Fix bug with MIME type warning on good labels

1. Add test resources : src/test/resources/github429
2. Add two missing types 'DOC' and 'DOCX' for 'Microsoft Word' and 'XLS' and 'XLSX' for 'Microsoft Excel' to mime table src/main/resources/validate_default_mime_types.txt
3. Add new regression test : src/test/resources/features/validate.feature

Refs:

https://github.com/nasa-pds/validate/issues/429 validate warns "document standard id ... is not correct" on good labels

Closes https://github.com/nasa-pds/validate/issues/429 validate warns "document standard id ... is not correct" on good labels

Several missing MIME types for "Microsoft Word" were added to validate_default_mime_types.txt to fix the issue: DOC and DOCX
Several missing MIME types for "Microsoft Excel" were added to validate_default_mime_types.txt to fix the issue: XLS and XLSX

Tested in DEV:

Running validate against a valid label with three document types '.DOC', '.TXT' and '.XLS'.  No warning messages are expected.

```
% validate -R pds4.label -r report_github429_label_valid.json   -s json -t  src/test/resources/github429/EPPS_EDR_SIS.xml
 
```
